windows-nsis/openvpn.nsi

```diff
@@ -30,6 +30,10 @@ SetCompressor lzma
 !addplugindir .
 !include "nsProcess.nsh"
 
+; DotNetChecker.nsh to detect whether .net 4.0 is enabled, which is required for openvpnserv2 ( https://github.com/ReVolly/NsisDotNetChecker )
+!addplugindir .
```

Comment:
It is added for safety. If we will ever remove nsProcess, this line must be kept